app/l10n.py

```diff
@@ -0,0 +1,12 @@
+from flask import request
+
+import os
+
+TRANSLATIONS = ['en'] + os.listdir('/noi/app/translations')
```

Comment:
errr, I'm doing this by listing the `translations` directory, but maybe I should be looking in the deployment config instead?